libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/reader/RtpH263Reader.java

```diff
@@ -190,11 +192,11 @@ private void getBufferFlagsAndResolutionFromVop(ParsableByteArray data, boolean
           height = (short) (144 << (sourceFormat - 2));
```

Comment:
Can you explain a bit here? I'm particularly lost why the shift bits are derived from a `sourceFormat`. Is the related logic in the H263 spec? I can't find it in RFC4629.